haskell/src/Monocle/Prelude.hs

```diff
@@ -435,6 +436,9 @@ getExn (Left err) = error (toText err)
 stripSpaces :: Text -> Text
 stripSpaces = T.replace " " ""
 
+removeTrailingSlash :: Text -> Text
+removeTrailingSlash = T.dropWhileEnd (== '/')
```

Comment:
I think there is a rule that says that if the name of function is as long as the expression, then it is not worth defining it... Perhaps this should be part of a `Text -> ApiURL` function?